src/core/auth.ts

```diff
@@ -23,8 +23,7 @@ export const auth = (client: ApolloClient<NormalizedCacheObject>): AuthSDK => {
   /**
    * Authenticates user with email and password.
    *
-   * @param email - User's email
-   * @param password - User's password
+   * @param opts - Object with user's email and password
```

Comment:
WDYT about adding argument type? https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html
Not sure if code editors can expand such info in meaningful way

Reply:
Correct me if I'm wrong but this link reffers to jsdoc syntax, with tsdoc syntax the type is inffered from `function sum(a: number): number` typing

Reply:
Okay! then LGTM